bundles/azure/aso/psql/bundle/config/01-flexible-server.ytt.yml

```diff
@@ -0,0 +1,248 @@
+#@ tags_list = []
+#@ tags_list.extend(data.values.global_tags)
+#@ tags_list.extend(data.values.database.tags)
+#@ tags = {x["key"]: x["value"] for x in tags_list}
+#@ if/end len(tags) > 0:
+  tags: #@ tags
+
+#@ fwRuleCounter = 0
+#@ for rule in data.values.firewall_rules:
+#@   startAddr = ip.parse_addr(rule.startIpAddress)
+#@   endAddr = ip.parse_addr(rule.endIpAddress)
+#@   tags_list = []
+#@   tags_list.extend(data.values.global_tags)
+#@   tags_list.extend(rule.tags)
+#@   tags = {x["key"]: x["value"] for x in tags_list}
+--- 
+apiVersion: dbforpostgresql.azure.com/v1beta20210601
+kind: FlexibleServersFirewallRule
+metadata:
+  name: #@ data.values.name + "-" + str(fwRuleCounter)
+  namespace: #@ data.values.namespace
+spec:
+  owner:
+    name: #@ flexibleServerName
+  startIpAddress: #@ startAddr.string()
+  endIpAddress: #@ endAddr.string()
+#@   if/end len(tags) > 0:
+  tags: #@ tags
+#@   fwRuleCounter += 1
+#@ end  
+---
+apiVersion: dbforpostgresql.azure.com/v1beta20210601
+kind: FlexibleServer
+metadata:
+  name: #@ flexibleServerName
+  namespace: #@ data.values.namespace
+spec:
+  location: #@ data.values.location
+  azureName: #@ flexibleServerName
+  owner:
+    name: #@ resourceGroupK8SName
+  version: #@ data.values.server.version
+  sku:
+    name: #@ data.values.server.instance_type
+    tier: #@ data.values.server.instance_tier
+  administratorLogin: #@ data.values.server.administrator_name
+  administratorLoginPassword:
+    name: #@ dbSecretName
+    key: password
+  storage:
+    storageSizeGB: #@ data.values.server.instance_storage_size_gb
+#@ tags_list = []
+#@ tags_list.extend(data.values.global_tags)
+#@ tags_list.extend(data.values.server.tags)
+#@ tags = {x["key"]: x["value"] for x in tags_list}
+#@ if/end len(tags) > 0:
+  tags: #@ tags
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: #@ serviceAccountName
+  namespace: #@ serviceAccountNamespace
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: #@ roleName
+  namespace: #@ data.values.namespace
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
+  resourceNames:
+  - #@ data.values.name
+- apiGroups:
+  - dbforpostgresql.azure.com
+  resources:
+  - flexibleservers
+  - flexibleserversdatabases
+  verbs:
+  - get
+  - list
+  - watch
+  resourceNames:
+  - #@ data.values.name
+- apiGroups:
+  - resources.azure.com
+  resources:
+  - resourcegroups
+  verbs:
+  - get
+  - list
+  - watch
+  resourceNames:
+  - #@ data.values.name
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: #@ roleName
+  namespace: #@ data.values.namespace
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: #@ roleName
+subjects:
+- kind: ServiceAccount
+  name: #@ serviceAccountName
+  namespace: #@ serviceAccountNamespace
+---
+apiVersion: secretgen.carvel.dev/v1alpha1
+kind: SecretTemplate
+metadata:
+  name: #@ data.values.name + "-bindable"
```

Comment:
_question_: This will be the name apps devs / operators will actually see. Not sure if this is a good name?

Reply:
As far as I can tall, its the same name used in the documentation of the existing packages. We chose to stick to that. It is documented as such as well I believe.

Reply:
just a _nit_ anyway.